addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -423,27 +423,25 @@
             <t t-set="can_access_channel" t-value="slide.channel_id.is_member or slide.channel_id.can_publish"/>
             <t t-if="can_access_channel">
                 <t t-set="links" t-value="slide.slide_resource_ids.filtered(lambda res: res.resource_type == 'url')"/>
-                <div class="col-12 col-md d-flex align-items-start mb-4 mb-md-0" t-if="links">
-                    <span class="text-muted fw-bold me-3">External sources</span>
-                    <div class="text-muted me-auto border-start ps-3">
+                <div class="d-flex mb-4 mt-4" t-if="links">
+                    <span class="text-muted fw-bold col-4 col-md-3">External sources</span>
+                    <div class="text-muted me-auto border-start ps-3 col-8 col-md-9">
                         <t t-foreach="links" t-as="link">
                             <a t-att-href="link.link" t-esc="link.name"/><br />
                         </t>
                     </div>
                 </div>
-                <group>
-                    <t t-set="files" t-value="slide.slide_resource_ids.filtered(lambda res: res.resource_type == 'file' and res.data)"/>
-                    <div class="col-12 col-md d-flex align-items-start mb-4 mb-md-0 o_wslides_js_course_join" t-if="files">
-                        <span class="text-muted fw-bold me-3">
-                            Additional Resources
-                        </span>
-                        <div class="text-muted me-auto border-start ps-3">
-                            <t t-foreach="slide.slide_resource_ids" t-as="resource" t-if="resource.resource_type == 'file' and resource.data">
-                                <a t-att-href="resource.download_url" t-esc="resource.name"/><br />
-                            </t>
-                        </div>
+                <t t-set="files" t-value="slide.slide_resource_ids.filtered(lambda res: res.resource_type == 'file' and res.data)"/>
+                <div class="d-flex mb-4 o_wslides_js_course_join" t-if="files">
```

Comment:
Same remark here.

Reply:
Hello @awa-odoo 

Earler i had fixed this issue by using the col class inside the col class.

But as per this discussion https://github.com/odoo/odoo/pull/116634#discussion_r1150656770  with TDE it is not ideal way to use col inside the col, here in this scenario also the parent div is row and if we use row in this div also it will contradict that point. what is your suggestion regarding this?

Thanks!!!!!

Reply:
Hello.

Can you remove "row" on the parent then, as we removed the col classes from its children anyway so it does not really make sense to keep it?

Reply:
> Hello.
> 
> Can you remove "row" on the parent then, as we removed the col classes from its children anyway so it does not really make sense to keep it?

Hi @awa-odoo 

Your suggestions had worked.

can you please review.

Thanks !!!